### PR TITLE
Removes vestiges of ./code-studio.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Note that I'm just deleting stuff "randomly", please look that the stuff I'm hacking away at should be hacked away.

File: aws/ci_build.rake [hunk 1]
```diff
@@ -138,16 +138,6 @@ task :apps_task do
...
-# TODO: This is a temporary task meant to cleanup old code-studio packages.
-# It should go away in the nearish future
-task :code_studio_task do
-  HipChat.log 'Cleaning code-studio package'
-  # get rid of any symlink to a built package
-  RakeUtils.system 'rm', dashboard_dir('public/code-studio') if File.exist?(dashboard_dir('public/code-studio'))
-  # also delete the package itself
-  RakeUtils.system 'rm -rf', dashboard_dir('public/code-studio-package') if File.exist?(dashboard_dir('public/code-studio-package'))
-end
-
```
Comment (Bjvanminnen) [inline aws/ci_build.rake:129]: I've been meaning to do this for a bit but haven't had a chance. The main thing I'd be worried about for this particular removal is that the right thing happens when creating an adhoc. I think in that case we start with an image that still has code-studio paths, and needs a cleanup step. Might be that part of the answer is to update our starting image.

File: lib/rake/build.rake [hunk 2]
```diff
@@ -56,19 +56,6 @@ namespace :build do
...
-  task :code_studio do
-    if File.exist?(code_studio_dir)
-      Dir.chdir(code_studio_dir) do
-        HipChat.log 'Removing <b>code-studio</b>...'
-        RakeUtils.system 'rm -rf build'
-      end
-    end
-  end
-  task :'code-studio' => :code_studio
-
```
Comment (Bjvanminnen) [inline lib/rake/build.rake:51]: same as comment in ci_build. I'm not sure which ends up being called as part of adhoc.

File: aws/ci_build.rake (hunk 1 above)
Comment (ashercodeorg) [reply aws/ci_build.rake:129]: If it is baked into the image, we should update the starting image. Any idea how to go about doing so?

**Comment by ashercodeorg** [comment]
It looks like CircleCI also has a step trying to build code studio. Any idea how to eliminate it?

**Comment by Bjvanminnen** [comment]
circle.yml has a couple of lines about adding code-studio related things to locals.yml, which can probably be removed. I wouldn't have expected it to be trying to build code-studio directly though. I dont know about the adhoc starting image.

**Comment by ashercodeorg** [comment]
Removed two lines from `circle.yml`. Going to ping wjordan later about adhoc creation.

**Comment by ashercodeorg** [comment]
At the risk of kicking the adhoc can (since I'm unable to create one from a clean branch), I'm going to merge.

**Pull request merged**